.github/workflows/integration.yml

```diff
@@ -6,13 +6,17 @@ jobs:
   integration:
     name: Integ
     runs-on: ubuntu-latest
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+      cancel-in-progress: true
     strategy:
       fail-fast: false
       matrix:
         cluster: ["opensearch"]
         secured: ["true"]
         entry:
           - { opensearch_version: 2.7.0 }
```

Comment:
I like this a lot! May be we can try to keep `2.9.0` & `2.11.0`

`2.9.0` --> GA (General Availability) for our model serving framework in ml-commons
`2.11.0` --> Current version


What do you think?

Reply:
Yeah, I am a bit new to github actions. Trying to see how to make it work that way.

Reply:
Good luck and thanks a lot for your thoughtful contribution.